projects/netgrif-components-core/src/lib/panel/task-panel-single/abstract-single-task.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, EventEmitter, Inject, Input, OnDestroy, Optional, Output, TemplateRef } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+import { TaskPanelData } from '../task-panel-list/task-panel-data/task-panel-data';
+import { MatExpansionPanel } from '@angular/material/expansion';
+import { HeaderColumn } from '../../header/models/header-column';
+import { TaskEventNotification } from '../../task-content/model/task-event-notification';
+import { LoggerService } from '../../logger/services/logger.service';
+import { NAE_TAB_DATA } from '../../tabs/tab-data-injection-token/tab-data-injection-token';
+import { InjectedTabData } from '../../tabs/interfaces';
+import { ActivatedRoute } from '@angular/router';
+
+
+@Component({
+    selector: 'ncc-abstract-single-task',
+    template: ''
+})
+export abstract class AbstractSingleTaskComponent implements OnDestroy {
```

Comment:
plz test

Reply:
This was already implemented, but somehow my git just went crazy and removed added files from uncommited changes. Pushing now.